features/steps/partcad-cli/commands/init.py

```
@@ -0,0 +1,134 @@
+    logging.debug(f"Command output: {result.stdout}")
+    logging.debug(f"Command error: {result.stderr}")
+
+    # Store the result in the context for further steps
+    context.result = result
+
+
+@then('a file named "{filename}" should be created')
+def step_impl(context, filename):
+    # Check if the file exists in the current directory
+    file_path = os.path.join(context.test_dir, filename)
+    assert os.path.isfile(file_path), f"File '{filename}' was not created"
```

Ruff (0.8.0)

Use of `assert` detected

(S101)